src/index.ts

```diff
@@ -1,4 +1,4 @@
 export * from "./addLeadingZeros";
 export * from "./durationToMilliseconds";
-export * from "./format";
+// export * from "./format";
```

Comment:
Why is this commented out?

Reply:
I don't want it to be exposed when published, but it will be once I develop it further.

Reply:
Fine bb.